src/envoy/mixer/http_control.cc

```diff
@@ -124,6 +125,12 @@ void FillRequestInfoAttributes(const AccessLog::RequestInfo& info,
     attr->attributes[kAttrNameResponseTime] =
         Int64Value(info.duration().count());
   }
+  if (info.responseCode().valid()) {
+    attr->attributes[kStatusCode] =
+        StringValue(std::to_string(info.responseCode().value()));
+  } else {
+    attr->attributes[kStatusCode] = StringValue("500");
```

Comment:
I just want to understand if a request is rejected by our "Check" call,  what is its responseCode().

If its responseCode() is not valid,  then, we have to set proper status.code for it.  

Actually, when Check() fails,  we did not set proper ResponseFlag.  So we may have to use our own flag in Instance class to remember it.